pl/mozorg/contribute/index.lang

```diff
@@ -242,156 +242,156 @@ Get Involved: Volunteer Opportunities at Mozilla
 
 
 ;To get started, tell us what interests you.
-To get started, tell us what interests you.
+Aby rozpocząć, powiedz nam, co cię interesuje.
 
 
 ;Choose one for now. You can always contribute to other areas later.
-Choose one for now. You can always contribute to other areas later.
+Wybierz jedno. Możesz pomóc w innych zakresach później.
 
 
 # This text is used for accessibility reasons, it's not visible
 ;Where would you like to get started?
-Where would you like to get started?
+Gdzie chciał(a)byś zacząć?
 
 
 ;Coding
-Coding
+Programowanie
 
 
 ;More about coding
-More about coding
+Więcej o programowaniu
 
 
 ;Contribute code to a whole range of Mozilla products, including Firefox and Firefox OS.
-Contribute code to a whole range of Mozilla products, including Firefox and Firefox OS.
+Twórz kod dla szerokiego zakresu produktów Mozilli, tj. Firefox i Firefox OS.
 
 
 ;Testing
-Testing
+Testowanie
 
 
 ;More about testing
-More about testing
+Więcej o testowaniu
 
 
 ;Help make the Mozilla products you use — and love — even better.
-Help make the Mozilla products you use — and love — even better.
+Pomóż nam czynić produkty Mozilli których używasz (i kochasz) jeszcze lepszymi.
 
 
 ;Writing
-Writing
+Pisanie
 
 
 ;More about writing
-More about writing
+Więcej o pisaniu
 
 
 ;Lend your voice to Mozilla — from blogging and social media to providing technical documents for users and developers.
-Lend your voice to Mozilla — from blogging and social media to providing technical documents for users and developers.
+Użycz swojego głosu Mozilli - od blogowania i portali społecznościowych do dostarczania dokumentów technicznych użytkownikom i programistom.
 
 
 ;Teaching
-Teaching
+Nauczanie
 
 
 ;More about teaching
-More about teaching
+Więcej o nauczaniu
 
 
 ;Share your skills with a community that’s eager to learn. Take advantage of our free tools, teaching resources and events.
-Share your skills with a community that’s eager to learn. Take advantage of our free tools, teaching resources and events.
+Podziel się swoimi umiejętnościami z chętną do nauki społecznością. Czerp korzyści z naszych bezpłatnych narzędzi, materiałów i wydarzeń.
 
 
 ;Helping
-Helping
+Pomoc
 
 
 ;More about helping
-More about helping
+Więcej o pomocy
 
 
 ;Provide support to users who need it and help everyone get the most out of their Mozilla products.
-Provide support to users who need it and help everyone get the most out of their Mozilla products.
+Dostarczanie wsparcia potrzebującym go użytkownikom i pomoc w czerpaniu z możliwości produktów Mozilli.
```

Comment:
„Dostarczanie” to kalka językowa.
Druga część zdania też jest trochę niezręczna, trzeba je przepisać.